cmd/fpga_crihook/main.go

```diff
@@ -44,54 +48,56 @@ func decodeJSONStream(reader io.Reader) (map[string]interface{}, error) {
 }
 
 type hookEnv struct {
-	bitStreamDir string
-	config       string
-	execer       utilsexec.Interface
+	bitStreamDir  string
+	config        string
+	execer        utilsexec.Interface
+	aFUIDTemplate string
```

Comment:
formatting...

Reply:
what's wrong with it? gofmt doesn't complain

Reply:
weird... I clearly see in my browser that the type names in the struct are not aligned, yet `git diff` shows no problems with that. Must be a browser bug. Disregard then.